py3status/modules/volume_status.py

```diff
@@ -18,6 +20,9 @@
         (default 10)
     channel: Alsamixer channel to track.
         (default 'Master')
+    color_muted: set to 1 to color 'format_muted' depending on the volume
+        percent.
+        (default 0)
```

Comment:
`color_xxx` is reserved and should not be used in this way.  It is for defining colors which should be accessed as `self.py3.COLOR_XXXX` in the module.